apps/server/node/service.ts

```
@@ -1188,8 +1201,9 @@ export class ServerService {
     const executor = task.executor
     switch (executor.kind) {
       case 'connector':
-        if (this.#connector == null) throw new ConnectorTaskError('connector.unavailable', 'The Connector request could not be completed.')
-        return await this.#connector.execute(
+        const connector = this.#resolveConnector()
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)